src/home/templates/partials/_table.html

```diff
@@ -14,8 +20,10 @@ <h2 id="serviceName" class="text-lg font-semibold">Service Name</h2>
         <p class="text-sm"><strong>Type:</strong> <span id="serviceType">Homeless</span></p>
         <p class="text-sm"><strong>Rating:</strong> <span id="serviceRating">4.5</span></p>
         <p class="text-sm"><strong>Distance:</strong> <span id="serviceDistance">1.2 miles</span></p>
-        <p class="text-sm mt-2" id="serviceDescription">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
         <a id="getDirections" href="#" target="_blank" rel="noopener noreferrer" class="inline-block bg-blue-500 text-white px-4 py-2 rounded hover:bg-blue-600 mt-2">Get Directions</a>
+        <div class="space-y-1 mb-4">
+            <div id="serviceDescription" class="text-sm mt-2"></div>
+        </div>
```

Comment:
adding a table within this container seems more intuitive. what do you think?

Reply:
Hmm, i would have to think about that. I initially made it this way because this was designed to handle an unknown amount of tables generated after looking through the json with the JS.

Reply:
ah okay, descriptions aren't consistent, so makes sense